lib/fluent/log.rb

```diff
@@ -426,7 +491,7 @@ def caller_line(type, time, depth, level)
                        else
                          "".freeze
                        end
-      log_msg = "#{time.strftime(@time_format)}[#{LEVEL_TEXT[level]}]: #{worker_id_part}"
+      log_msg = "#{time.strftime(@time_format)} [#{LEVEL_TEXT[level]}]: #{worker_id_part}"
```

Comment:
Can't this log pattern be customizable as well?
For example, it may be necessary to have patten like `[time][log_level] message`.

It could be a customizable part of `text` format or a new format, let's say `custom`.